### PR TITLE
feature/VETHUB-220

### PR DESCRIPTION
**Pull request by Jo-Zh**
-implement styling and responsive css
-create collapse btn component
To test: `yarn test`
Currently available in `testsso-uat` zone

image

### CODE REVIEW

**Comment by anzau** [comment]
I like what you've done there, although it feels a bit odd with the wording, maybe use the icon for both open and closing and remove the white background?

image

File: src/components/CrucibleFilter.vue [hunk 1]
```diff
@@ -1,78 +1,98 @@
...
-    }
-  });
 };
 
 // Filter tags send to back end to filter resource
 const getFilterTag = (key: string, tag: string) => {
-  showDropdown.value[key] = !showDropdown.value[key];
   if (!filterTagArray.value.includes(`${key}:${tag.replace(" ", "_")}`)) {
     filterTagArray.value.push(`${key}:${tag.replace(" ", "_")}`);
   }
```
Comment (freshc0w) [inline src/components/CrucibleFilter.vue:89]: I think in terms of readability, assigning a variable here would be a bit better. i.e.
```ts
  const formattedTag = `${key}:${tag.replace(" ", "_")}`;
  if (!filterTagArray.value.includes(formattedTag)) {
    filterTagArray.value.push(formattedTag);
  }
```

**Comment by Jo-Zh** [comment]
fixed

File: src/components/FilterButton.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  background-color: transparent !important;
+  color: #49075e;
+  cursor: pointer;
+}
+
+.filter-btn:hover {
+  background-color: #49075e !important;
+  color: white !important;
+}
+</style>
```
Comment (freshc0w) [inline src/components/FilterButton.vue:44]: Using `!important` is considered very bad practice in css and is considered bad practice. Use it sparingly.
These `!important` can be removed and replaced by adding `<style scoped>` at the top

**Review by freshc0w — APPROVED** [review]
See suggestions. Really recommend removing the !important.
Otherwise LGTM

File: src/components/CrucibleFilter.vue (hunk 1 above)
Comment (Jo-Zh) [reply src/components/CrucibleFilter.vue]: fixed

**Pull request merged**